### DISCUSSION TITLE
Help: I cannot find this missing icon (codepoint: \u{e975}).

### DISCUSSION THREAD

**Discussion by xzhang8102** [LazyVim/LazyVim#4559] [Q&A] [answered]
When I opened up lazygit popup, I got this warning message: "No fonts contain glyphs for these codepoints: \u{e975}. Placeholder glyphs are being displayed instead.".

![image](https://github.com/user-attachments/assets/3331c303-8030-4e96-97ff-12522561a802)

I searched issues in serveral relatable repos, but I still cannot figure out how to resolve this. Maybe someone could give me a hand?

Terminal: wezterm
Font: Fira Code Nerd Font

**Comment by xzhang8102** [comment]
Relatable issue: https://github.com/jesseduffield/lazygit/issues/3747#issue-2407423220

**Reply by mark2185** [reply]
Does the same happen to you when launching `lazygit` from `wezterm` directly?

**Reply by xzhang8102** [reply, marked as answer]
Yes, it seems a lazygit issue.
